subprojects/docs/src/snippets/ivy-publish/java-multi-project/kotlin/project1/build.gradle.kts

```diff
@@ -0,0 +1,10 @@
+plugins {
+    id("myproject.publishing-conventions")
+}
+
+description = "The first project"
+
+dependencies {
+    "implementation"("junit:junit:4.13")
+    "implementation"(project(":project2"))
```

Comment:
:x: These should now be `implementation` (without `"`) as the accessors are now available.